cmake/FreecivInstall.cmake

```diff
@@ -160,6 +160,13 @@ if(UNIX AND NOT APPLE)
     message(STATUS "CMAKE_INSTALL_PREFIX changed from the default to /usr.")
   endif()
 
+  # Generate the manual pages with Sphinx
+  if(SPHINX_FOUND)
+    message(STATUS "Creating manual pages...")
+    execute_process(
+        COMMAND sphinx-build -b man ${CMAKE_SOURCE_DIR}/docs ${CMAKE_BINARY_DIR}/man)
```

Comment:
Does this run at configure time or at build time? I would prefer build time...

Reply:
It seems to run at configure time, even though the code is in the install section. So it really should run at build or install time, but not sure how to tell cmake to do that.

Reply:
The CMake interface to execute commands at build time is through [`add_custom_command`](https://cmake.org/cmake/help/latest/command/add_custom_command.html) and [`add_custom_target`](https://cmake.org/cmake/help/latest/command/add_custom_target.html).

Reply:
but... I don't think we need a custom command or target and adding one seems like overkill